docs/how-to/simple-authpolicy-for-app-developers.md

```diff
@@ -0,0 +1,254 @@
+# Authenticated Rate Limiting for Application Developers
```

Comment:
This walkthrough tackles auth and RL at once. Worth perhaps having another (much simpler) one to cover only the ext-auth part (i.e., no RL), and then maybe this one would come after? So the order becomes: (i) Multi-cluster stuff, (ii) RL, (iii) Ext-auth, (iv) Authenticated-RL. Arguably, (ii) to (iv) could all be about targeting the gateway instead of HTTPRoutes.

Reply:
Agreed - I thought about a similar order as well